packages/wallet/src/services/WalletUtil.ts

```diff
@@ -26,7 +26,7 @@ export interface WalletOutputValidatorContext {
 }
 
 export type WalletUtilContext = WalletOutputValidatorContext &
-  InputResolverContext & { chainHistoryProvider: ChainHistoryProvider };
+  InputResolverContext & { chainHistoryProvider?: ChainHistoryProvider };
```

Comment:
Where is this used in Lace? Are we sure we don't need 'backend input resolver' there?

Reply:
We dont need it, I added it in my plutus script PR, but made it requiered, and LAce doesnt have the chainHistoryProvider where it calls this function. Would be nice to have, but I didnt see an obvious way to get the provider since they have at those points only an observable wallet (rather than a base wallet). So I made the argument optional